content/millennium/r4/base/entities.md

```diff
@@ -7,6 +7,6 @@ layout: api
 
 <%= render '/millennium/r4/base/entities.*' %>
 
-* [Organization](../base/entities/organization)
-* [Location](../base/entities/location)
-* [Device](../base/entities/device)
+* [Organization](../entities/organization)
+* [Location](../entities/location)
+* [Device](../entities/device)
```

Comment:
I think these links should be like this https://github.com/cerner/fhir.cerner.com/pull/710/files#diff-db70aef4601cdc7697917f05be2110c0c9e05c4346f1e0c64a137e344b3d3bd6R32 to avoid relative paths. Need to apply throughout PR

Reply:
[commit](https://github.com/cerner/fhir.cerner.com/pull/710/commits/649d44c5f091f407f92d3a21a9e14737f62249ea)